src/gh.js

```diff
@@ -25,7 +25,7 @@ async function getRunner(label) {
 }
 
 async function waitForRunnerRegistered(label) {
-  const timeoutSeconds = 900
+  const timeoutSeconds = 1800
```

Comment:
any reason for this change ?

Reply:
Yes, hyperstack instance can be super slow finish their initialization phase.